RecoTracker/ConversionSeedGenerators/python/ConversionStep_cff.py

```diff
@@ -244,8 +244,8 @@
     )
 )
 
-photonConvTrajSeedFromSingleLeg.TrackRefitter = cms.InputTag('generalTracks')
-photonConvTrajSeedFromSingleLeg.primaryVerticesTag = cms.InputTag('firstStepPrimaryVertices')
+photonConvTrajSeedFromSingleLeg.TrackRefitter      = 'generalTracks'
+photonConvTrajSeedFromSingleLeg.primaryVerticesTag = 'firstStepPrimaryVertices'
 #photonConvTrajSeedFromQuadruplets.TrackRefitter = cms.InputTag('generalTracks')
```

Comment:
Since you are there, you can complete the job by updating also these two lines, even if commented out
Beside that, the PR is ready to go